.github/workflows/uc_loadsim.yml

```diff
@@ -30,9 +37,528 @@ defaults:
+        echo "${{ secrets.DIGICERT_KEY }}" | base64 -d > key.pem
+
+    - name: Save SSH key for access to EC2 instance
+      run: |
+        echo "${{ secrets.SSH_LOADSIM }}" > ssh_key.pem
+        chmod 600 ssh_key.pem
+
+    - name: Try connecting to EC2 instance
+      run: |
+        until ssh -o "StrictHostKeyChecking no" -i ssh_key.pem ubuntu@${{ steps.ec2_instance.outputs.instance_public_ip }}; do
+            sleep 5
+        done
+        ssh -i ssh_key.pem ubuntu@${{ steps.ec2_instance.outputs.instance_public_ip }} << EOF
+          sudo -i;
+          whoami;
+          curl https://get.docker.com -o docker.sh;
+          chmod +x docker.sh;
+          bash docker.sh;
+          apt install -y python3-pip jq git;
+          pip3 install docker-compose;
+        EOF
+        scp -i ssh_key.pem cert.pem ubuntu@${{ steps.ec2_instance.outputs.instance_public_ip }}:/tmp/websocket-cert.pem
+        scp -i ssh_key.pem key.pem ubuntu@${{ steps.ec2_instance.outputs.instance_public_ip }}:/tmp/websocket-key.pem
+
+    - name: Deploy target CloudSDK in EC2
+      run: |
+        ssh -o "StrictHostKeyChecking no" -i ssh_key.pem ubuntu@${{ steps.ec2_instance.outputs.instance_public_ip }} << EOF
+          sudo -i;
+          git clone https://github.com/Telecominfraproject/wlan-cloud-ucentral-deploy.git;
+          cd wlan-cloud-ucentral-deploy;
+          git checkout ${{ github.event.inputs.sdk_version }};
+          cd docker-compose;
+          sed "s/#SIMULATORID=/SIMULATORID=$SIMULATORID/" -i owgw.env;
+          cp /tmp/websocket-cert.pem certs/websocket-cert.pem;
+          cp /tmp/websocket-key.pem certs/websocket-key.pem;
+          docker-compose up -d
+        EOF
+
+    # Deploy CloudSDK with OWLS in K8s
+    - name: Checkout Testing repo
+      uses: actions/checkout@v2
+      with:
+        path: wlan-testing
+
+    - name: Get EKS access credentials
+      run: aws eks update-kubeconfig --name ${{ env.AWS_EKS_NAME }}
+
+    - name: Set gateway URL output
+      id: gateway
+      run: echo "::set-output name=url::gw-${{ steps.ec2_facts.outputs.instance_name }}.cicd.lab.wlan.tip.build"
+
+    - name: Set sec service URL output
+      id: sec
+      run: echo "::set-output name=url::sec-${{ steps.ec2_facts.outputs.instance_name }}.cicd.lab.wlan.tip.build"
+
+    - name: Set ls service URL output
+      id: ls
+      run: echo "::set-output name=url::ls-${{ steps.ec2_facts.outputs.instance_name }}.cicd.lab.wlan.tip.build"
+
+    - name: Prepare certificates from secrets
+      working-directory: wlan-testing/helm/ucentral
+      run: |
+        echo "${{ secrets.DIGICERT_CERT }}" | base64 -d > cert.pem
+        echo "${{ secrets.DIGICERT_KEY }}" | base64 -d > key.pem
+        echo "${{ secrets.LOADSIM_DIGICERT_CERT }}" | base64 -d > device-cert.pem
+        echo "${{ secrets.LOADSIM_DIGICERT_KEY }}" | base64 -d > device-key.pem
+
+    - name: Deploy CloudSDK instance with load simulator
```

Comment:
I think you mentioned this but I don't remember anymore: why is a complete CloudSDK instance + OWLS deployed in Kubernetes at this point instead of just a plain OWLS deployment on EC2?

Reply:
I've decided to go with this approach to decrease amount of deployment scenarios - with current approach we are using only one method for OWLS deployment which is Helm deployment in the whole stack and I'm not sure if one more deployment method will make everything easier for a future maintenance